calcos/concurrent.py

```diff
@@ -864,7 +864,7 @@ def shift1Corr(self, n, i0, i1, extrapolate=False):
         for segment in self.segment_list:
 
             # Restrict the correction to the applicable regions.
-            shift_flags = np.zeros(i1 - i0, dtype=np.bool8)
+            shift_flags = np.zeros(i1 - i0, dtype=np.bool_)
```

Comment:
Is it necessary to use numpyp scalars or would `bool` be sifficient? I know in pther packages we resorted to using the Python `bool` but there might be a good reason to use `np._bool`.

Reply:
I didn't really think very hard about whether to use bool or np.bool_, I just followed the prescription passed by the deprecation warning for how to fix the use of np.bool8